Formula/whatmp3.rb

```diff
@@ -29,7 +29,11 @@ def install
 
   test do
     # Create dummy FLAC
-    (testpath/"flac/file.flac").write "fLaC\x00\x00\x00\"\x04\x80\x04\x80\x00\x00\f\x00\x00\f\x01\xF4\x00\xF0\x00\x00\x00\x01\xF3\x8B\xE3\xDBM\x93\xE40\\~$\xBE\x94\xEF\x01\x9A\x84\x00\x00( \x00\x00\x00reference libFLAC 1.2.1 20070917\x00\x00\x00\x00\xFF\xF8d\b\x00\x00\xE3\x03\x01\xFD\xEC\x10"
+    (testpath/"flac/file.flac").write \
```

Comment:
Can we find a way to avoid backslash continuations please

Reply:
_Please_ 😧.

Reply:
Okay, that comment can be read two ways. I was joining in with ILZ's desire to avoid backslashes, for clarification 🙈.

Reply:
We all know of your hatred of backslash continuations so there was really only one possible reading.

Reply:
Y'all also know my love of being occasionally sarcastic so I thought I'd cover myself.

Reply:
It's (long) string without newlines. What do you suggest? I agree with @reitermarkus that the "`EOS` and strip the newlines" approach isn't a nice one; particularly here where it's not even prose but hex data.

Reply:
In short: when you need a long string in Ruby but don't want to have incredibly long lines: this is how you do it. The other alternatives are all significantly more cumbersome in terms of syntax. I appreciate it's not ideal.

Reply:
I suggest we not apply this rule.

Reply:
For consistency, all of these would need to be changed, too, because the only reason they were allowed to run over 80 characters was to keep them on one line. Just changing one of them because some arbitrary rubocop rule fell out the sky doesn't make any sense to me. But I don't think this rule is buying us anything given how few "violations" there are.
```
81	influxdb	  plist_options :manual => "influxd -config #{HOMEBREW_PREFIX}/etc/influxdb.conf"
82	cassandra@2.1	  plist_options :manual => "#{HOMEBREW_PREFIX}/opt/cassandra@2.1/bin/cassandra -f"
82	influxdb@0.8	  plist_options :manual => "influxdb -config=#{HOMEBREW_PREFIX}/etc/influxdb.conf"
82	postgresql@9.4	  plist_options :manual => "pg_ctl -D #{HOMEBREW_PREFIX}/var/postgresql@9.4 start"
82	postgresql@9.5	  plist_options :manual => "pg_ctl -D #{HOMEBREW_PREFIX}/var/postgresql@9.5 start"
82	postgresql@9.6	  plist_options :manual => "pg_ctl -D #{HOMEBREW_PREFIX}/var/postgresql@9.6 start"
82	supervisor	  plist_options :manual => "supervisord -c #{HOMEBREW_PREFIX}/etc/supervisord.ini"
82	telegraf	  plist_options :manual => "telegraf -config #{HOMEBREW_PREFIX}/etc/telegraf.conf"
84	kettle	  plist_options :manual => "pdicarte #{HOMEBREW_PREFIX}/etc/kettle/carte-config.xml"
84	mysql@5.5	  plist_options :manual => "#{HOMEBREW_PREFIX}/opt/mysql@5.5/bin/mysql.server start"
84	mysql@5.6	  plist_options :manual => "#{HOMEBREW_PREFIX}/opt/mysql@5.6/bin/mysql.server start"
85	kapacitor	  plist_options :manual => "kapacitord -config #{HOMEBREW_PREFIX}/etc/kapacitor.conf"
86	lighttpd	  plist_options :manual => "lighttpd -f #{HOMEBREW_PREFIX}/etc/lighttpd/lighttpd.conf"
87	mariadb@10.0	  plist_options :manual => "#{HOMEBREW_PREFIX}/opt/mariadb@10.0/bin/mysql.server start"
87	mariadb@10.1	  plist_options :manual => "#{HOMEBREW_PREFIX}/opt/mariadb@10.1/bin/mysql.server start"
89	mosquitto	  plist_options :manual => "mosquitto -c #{HOMEBREW_PREFIX}/etc/mosquitto/mosquitto.conf"
89	olsrd	  plist_options :startup => true, :manual => "olsrd -f #{HOMEBREW_PREFIX}/etc/olsrd.conf"
90	artifactory	  plist_options :manual => "#{HOMEBREW_PREFIX}/opt/artifactory/libexec/bin/artifactory.sh"
93	gitbucket	  plist_options :manual => "java -jar #{HOMEBREW_PREFIX}/opt/gitbucket/libexec/gitbucket.war"
94	nagios	  plist_options :startup => true, :manual => "nagios #{HOMEBREW_PREFIX}/etc/nagios/nagios.cfg"
97	datomic	  plist_options :manual => "transactor #{HOMEBREW_PREFIX}/etc/datomic/free-transactor.properties"
102	stubby	  plist_options :startup => true, :manual => "sudo stubby -C #{HOMEBREW_PREFIX}/etc/stubby/stubby.yml"
104	collectd	  plist_options :manual => "#{HOMEBREW_PREFIX}/sbin/collectd -f -C #{HOMEBREW_PREFIX}/etc/collectd.conf"
109	ooniprobe	  plist_options :startup => "true", :manual => "ooniprobe -i #{HOMEBREW_PREFIX}/share/ooniprobe/current.deck"
119	wildfly-as	  plist_options :manual => "#{HOMEBREW_PREFIX}/opt/wildfly-as/libexec/bin/standalone.sh --server-config=standalone.xml"
130	apache-archiva	  plist_options :manual => "ARCHIVA_BASE=#{HOMEBREW_PREFIX}/var/archiva #{HOMEBREW_PREFIX}/opt/apache-archiva/bin/archiva console"
130	derby	  plist_options :manual => "DERBY_OPTS=-Dsystem.derby.home=#{HOMEBREW_PREFIX}/var/derby #{HOMEBREW_PREFIX}/bin/startNetworkServer"
131	shadowsocks-libev	  plist_options :manual => "#{HOMEBREW_PREFIX}/opt/shadowsocks-libev/bin/ss-local -c #{HOMEBREW_PREFIX}/etc/shadowsocks-libev.json"
132	cayley	  plist_options :manual => "cayley http --assets=#{HOMEBREW_PREFIX}/share/cayley/assets --config=#{HOMEBREW_PREFIX}/etc/cayley.conf"
138	rtags	  plist_options :manual => "#{HOMEBREW_PREFIX}/bin/rdm --verbose --inactivity-timeout=300 --log-file=#{HOMEBREW_PREFIX}/var/log/rtags.log"
143	vnstat	  plist_options :startup => true, :manual => "#{HOMEBREW_PREFIX}/opt/vnstat/bin/vnstatd --nodaemon --config #{HOMEBREW_PREFIX}/etc/vnstat.conf"
147	pcap_dnsproxy	  plist_options :startup => true, :manual => "sudo #{HOMEBREW_PREFIX}/opt/pcap_dnsproxy/bin/Pcap_DNSProxy -c #{HOMEBREW_PREFIX}/etc/pcap_dnsproxy/"
150	mongo-orchestration	  plist_options :startup => true, :manual => "#{HOMEBREW_PREFIX}/opt/mongo-orchestration/bin/mongo-orchestration -b 127.0.0.1 -p 8889 --no-fork start"
169	kafka	  plist_options :manual => "zookeeper-server-start #{HOMEBREW_PREFIX}/etc/kafka/zookeeper.properties & kafka-server-start #{HOMEBREW_PREFIX}/etc/kafka/server.properties"
188	varnish@4	  plist_options :manual => "#{HOMEBREW_PREFIX}/sbin/varnishd -n #{HOMEBREW_PREFIX}/var/varnish -f #{HOMEBREW_PREFIX}/etc/varnish/default.vcl -s malloc,1G -T 127.0.0.1:2000 -a 0.0.0.0:8080"
191	varnish	  plist_options :manual => "#{HOMEBREW_PREFIX}/sbin/varnishd -n #{HOMEBREW_PREFIX}/var/varnish -f #{HOMEBREW_PREFIX}/etc/varnish/default.vcl -s malloc,1G -T 127.0.0.1:2000 -a 0.0.0.0:8080 -F"
324	grafana	  plist_options :manual => "grafana-server --config=#{HOMEBREW_PREFIX}/etc/grafana/grafana.ini --homepath #{HOMEBREW_PREFIX}/share/grafana cfg:default.paths.logs=#{HOMEBREW_PREFIX}/var/log/grafana cfg:default.paths.data=#{HOMEBREW_PREFIX}/var/lib/grafana cfg:default.paths.plugins=#{HOMEBREW_PREFIX}/var/lib/grafana/plugins"
```

Reply:
The status quo is files with poor readability and users who are unable to adequately be guided to good style. I'm not bulk changing all these in a oner because I'm trying to reduce the maximum lines bit by bit.

We could consider just using `EOS.undent` or something for those lines, handling it in Homebrew/brew so that they are output nicely or again whitelisting just the `plist_options` like `url` and `mirror` so we don't care about the length here.

Reply:
Although, to me, even the way the lines are displayed above helps to demonstrate why I don't think the status quo is great as the horizontal scrolling and/or wrapping required makes them unnecessarily hard to read. If we can get the rest of the files bar `url`s and `plist_options` sorted, though, then I'd rather that than nothing.

Reply:
>I'm not bulk changing all these in a oner because I'm trying to reduce the maximum lines bit by bit.

Carving it up based on some limit we keep fiddling with, as opposed to uniformly applying a rule to the plist_options component, is just chaos.